From 3356dece852805a24770836fe88553e1cf387d2c Mon Sep 17 00:00:00 2001
From: Vagrant Cascadian <vagrant@reproducible-builds.org>
Date: Wed, 5 Feb 2020 06:26:40 +0000
Subject: [PATCH] Convert to using automake: Switch Build-Depends from
 automake-1.15 to automake. Add dh_autoreconf before dh_auto_configure. Add
 dh_autoreconf_clean to clean target.

---
 debian/control | 2 +-
 debian/rules   | 2 ++
 2 files changed, 3 insertions(+), 1 deletion(-)

diff --git a/debian/control b/debian/control
index 37db203..25fd51e 100644
--- a/debian/control
+++ b/debian/control
@@ -2,7 +2,7 @@ Source: cabextract
 Section: utils
 Priority: optional
 Maintainer: Eric Sharkey <sharkey@debian.org>
-Build-Depends: debhelper (>= 11), sharutils, libmspack-dev, pkg-config, automake-1.15
+Build-Depends: debhelper (>= 11), sharutils, libmspack-dev, pkg-config, automake, dh-autoreconf
 Standards-Version: 4.2.1
 
 Package: cabextract
diff --git a/debian/rules b/debian/rules
index 9eafdb1..6fa5bbd 100755
--- a/debian/rules
+++ b/debian/rules
@@ -12,6 +12,7 @@ CPPFLAGS = `dpkg-buildflags --get CPPFLAGS`
 configure: configure-stamp
 configure-stamp:
 	dh_testdir
+	dh_autoreconf
 	# Add here commands to configure the package.
 	dh_auto_configure -- CPPFLAGS="$(CPPFLAGS)" CFLAGS="$(CFLAGS)" LDFLAGS="$(LDFLAGS)" --with-external-libmspack=yes
 	
@@ -32,6 +33,7 @@ build-stamp: configure-stamp
 	touch build-stamp
 
 clean:
+	dh_autoreconf_clean
 	dh_testdir
 	dh_testroot
 	rm -f build-stamp configure-stamp
-- 
2.20.1

